### PR TITLE
Update change log

### LINKED ISSUE #190: draft-05 chores
- [ ] Bump VERSION
- [ ] Regenerate test vectors
- [ ]  Update Change Log

### PR DESCRIPTION
**Pull request by cjpatton**
Closes #190.
Based on #195 (merge that first)

### CODE REVIEW

**Comment by cjpatton** [comment]
Rebased.

File: draft-irtf-cfrg-vdaf.md [hunk 1]
```diff
@@ -330,6 +330,14 @@ security considerations for VDAFs.
 
 05:
 
+* IdpfPoplar: Replace PrgSha3 with PrgFixedKeyAes128, a fixed-key mode for
+  AES-128 based on a construction from {{GKWWY20}}. This change is intended to
+  improve performance of IPDF evaulation. Note that the new PRG is not suitable
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:335]: nit: spelling
```suggestion
  improve performance of IDPF evaluation. Note that the new PRG is not suitable
```

**Review by divergentdave — APPROVED** [review]

File: draft-irtf-cfrg-vdaf.md (hunk 1 above)
Comment (cjpatton) [reply draft-irtf-cfrg-vdaf.md:335]: 👍

**Review by bifurcation — APPROVED** [review]

**Comment by cjpatton** [comment]
Squashed.

**Pull request merged**